prediction/classify/__init__.py

```diff
@@ -0,0 +1,4 @@
+import sys
+import os
+
+sys.path.append(os.path.join(os.path.dirname(__file__), os.path.pardir))
```

Comment:
Wait, what? :p

Reply:
As it was mentioned in the technical details from the [issue #2](https://github.com/concept-to-clinic/concept-to-clinic/issues/2): 
>  This feature should be implemented in the prediction/classify/trained_model/predict method. Code to train the model should live in the prediction/classify/src/ folder.   
> A fully serialized version of the model that can be loaded should live in the prediction/classify/assets/ folder using git-lfs.   

The `classify` shouldn't lie in the `prediction/src` directory but in the `prediction` itself. Is there a typo in the issue or is it now outdated?